celements-keycloak/src/main/java/com/celements/keycloak/KeycloakService.java

```diff
@@ -79,21 +149,41 @@ public AuthenticationManager getAuthenticationManagerForWiki(WikiReference wikiR
         this::buildAuthenticationManagerForWiki);
   }
 
-  private AuthenticationManager buildAuthenticationManagerForWiki(String wikiName) {
+  @Override
+  @NotNull
+  public JwtDecoder getJwtDecoder() {
+    return jwtDecoderCache.computeIfAbsent(getWikiName().orElseThrow(),
+        this::buildJwtDecoderForWiki);
+  }
+
+  private JwtDecoder buildJwtDecoderForWiki(String wikiName) {
```

Comment:
```suggestion
  private JwtDecoder buildJwtDecoderForCurrentWiki() {
```
arg `wikiName` is not actually used for building